src/message.js

```diff
@@ -140,41 +141,56 @@ Message.prototype.decrypt = async function(privateKey, sessionKey, password) {
 };
 
 /**
- * Decrypt an encrypted session key either with a private key or a password.
- * @param  {Key} privateKey    (optional) private key with decrypted secret data
- * @param  {String} password   (optional) password used to decrypt
+ * Decrypt an encrypted session key either with private keys or passwords.
+ * @param  {Array<Key>} privateKeys    (optional) private key with decrypted secret data
+ * @param  {Array<String>} passwords   (optional) password used to decrypt
  * @return {Array<{ data:Uint8Array, algorithm:String }>} array of object with potential sessionKey, algorithm pairs
  */
-Message.prototype.decryptSessionKeys = function(privateKey, password) {
+Message.prototype.decryptSessionKeys = function(privateKeys, passwords) {
   var keyPackets = [];
   return Promise.resolve().then(async () => {
-    if (password) {
+    if (passwords) {
       var symESKeyPacketlist = this.packets.filterByTag(enums.packet.symEncryptedSessionKey);
       if (!symESKeyPacketlist) {
         throw new Error('No symmetrically encrypted session key packet found.');
       }
       await Promise.all(symESKeyPacketlist.map(async function(packet) {
-        try {
-          await packet.decrypt(password);
-          keyPackets.push(packet);
-        } catch (err) {}
+        for (var i = 0; i < passwords.length; i++) {
```

Comment:
Could do a 'await Promise.all(passwords.map(async function' thing like above and get rid of the eslint exception